### ISSUE TITLE
Question about normalize during inference(attack) phase

### ISSUE THREAD

**Issue by Gavin-1995** [Harry24k/adversarial-attacks-pytorch#63]
Hi author(s),

I have a question about the normalize operation with torchattacks.

During the training phase, I use the normalize operation on the training data, for example,

-----------------------------------------------------------------------------------------------------------
 transform_train = transforms.Compose([
    transforms.RandomCrop(32, padding=4),
    transforms.RandomHorizontalFlip(),
    transforms.ToTensor(),
    transforms.Normalize((0.4914, 0.4822, 0.4465), (0.2023, 0.1994, 0.2010)),
])
------------------------------------------------------------------------------------------------------------
Therefore, I need to perform the normalize operation during the inference (test, attack) phase. 
However, in torchattacks, the input data must be in the range of [0, 1].
So how to deal with this problem?

Best wishes,
Gavin

**Comment by Harry24k** [comment]
You can use Normalize class in front of the model.
Please refer to https://github.com/Harry24k/adversarial-attacks-pytorch/blob/master/demos/White%20Box%20Attack%20(ImageNet).ipynb.
```
class Normalize(nn.Module):
    def __init__(self, mean, std) :
        super(Normalize, self).__init__()
        self.register_buffer('mean', torch.Tensor(mean))
        self.register_buffer('std', torch.Tensor(std))
        
    def forward(self, input):
        # Broadcasting
        mean = self.mean.reshape(1, 3, 1, 1)
        std = self.std.reshape(1, 3, 1, 1)
        return (input - mean) / std
```
